source/common/runtime/runtime_features.cc

```diff
@@ -47,6 +47,13 @@ constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
     "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
+    "envoy.deprecated_features.config_source.proto:UNSUPPORTED_REST_LEGACY",
+    "envoy.deprecated_features.ext_authz.proto:use_alpha",
+    "envoy.deprecated_features.route.proto:enabled",
+    "envoy.deprecated_features.tcp_proxy.proto:deprecated_v1",
```

Comment:
istio might be a current user of  tcp_proxy.DeprecateV1. Could you hold on?

Reply:
Link? I couldn't find any references to it last time I looked (few weeks ago).

Reply:
Sorry, I don't find it either. Update my original comment